app/src/config/api-config.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * @author Shiv Kumar Ganesh <shivkumar.ganesh@sjsu.edu>
+ */
+export const API_URL = {
+    CREATE_TWEET : '/api/create',
+    DELETE_TWEET: '',
+    GET_TWEETS: '',
```

Comment:
Do we need a separate endpoint for get vs retrieve vs list tweets? Does this mean a particular tweet might be like clickable to retrieve one?

Reply:
Let it be there. I was thinking that if someone clicked on tweet do we pull information and edit or update an existing tweet.

Reply:
I think get is the same as the features of retrieve. 
This is our requirement from this HW "Develop a simple twitter service that implement any Twitter API to to programmatically create, retrieve and delete a Tweet in the Twitter Sandbox environment."